backend/main.py

```
@@ -234,45 +230,39 @@ async def create_issue(
 
         # Offload blocking file I/O to a thread
         def save_file():
-            with open(image_path, "wb") as buffer:
+            with open(file_location, "wb") as buffer:
                 shutil.copyfileobj(image.file, buffer)
 
         await asyncio.to_thread(save_file)
```

Ruff (0.15.20)

[error] Undefined name `asyncio`

(F821)